src/data-workspace/data-details-sidebar/update-limits.js

```diff
@@ -47,6 +49,7 @@ function UpdateLimits({
                         {...minField.input}
                         label={limitInputLabelsByName.min}
                         error={submitFailed && !!minField.meta.error}
+                        disabled={!canAdd}
```

Comment:
technically you can't get to the edit state if you don't have add authority, but I kept this check in case we wanted to change the logic/let user with delete authority open the "edit" window to perform the delete alone